### DISCUSSION TITLE
Allow customisation of address card output

### DISCUSSION THREAD

**Discussion by nfourtythree** [craftcms/cms#12615] [Ideas] [answered]
Allow the ability for developers to hook into the output of address cards in the CP.

This can be useful if it would be helpful to see one of the custom fields when viewing an address at a quick look in its card format.

### Related issues

craftcms/commerce#3082

**Comment by brandonkelly** [comment, marked as answer]
This is resolved for Craft 4.5 via a new `formatter` property on the `Addresses` service, which can be used to override the default address formatter. (#13242)
